### PR TITLE
add virsh vnc instructions

### PR DESCRIPTION
**Pull request by tahabi**

### CODE REVIEW

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 1]
```diff
@@ -138,3 +138,30 @@ until the configuration converges.
...
+port the VNC is listening on, via, e.g.:
+
+    ssh -L 5900:localhost:5905 <hypervisor>
+
+Finally, in yet another terminal, launch a VNC display to view the console by doing:
+
+    vncviewer localhost:5900
+
+This will open a VNC window that will allow you to monitor and interact with the
+remote console.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: These instructions seem really complicated, can't you just replace this section with these two lines?

```
me@desktop:~$ ssh -X jaws
me@jaws:~$ sudo -E virt-viewer death

Comment (tahabi) [reply ocfweb/docs/docs/staff/procedures/new-host.md]: nice, done

File: ocfweb/docs/docs/staff/procedures/new-host.md [hunk 2]
```diff
@@ -138,3 +138,13 @@ until the configuration converges.
 The first Puppet run and various other things may be broken if one or more
 packages are out of date, e.g. Puppet. Remedy this with an `apt-get update &&
 apt-get upgrade`.
+
+## Tips and Tricks
+
+### VNC monitoring of Debian install
```
Comment (jvperrin) [inline ocfweb/docs/docs/staff/procedures/new-host.md]: Might be useful to mention `virt-manager` here, but it does require the root password to use, so it's not the easiest, probably better to encourage `virt-viewer` instead.

**Comment by jvperrin** [comment]
Did you close this in favor of incorporating it into #323? I think it was fine as it was

**Comment by tahabi** [comment]
yeah

**Comment by jvperrin** [comment]
Ok, that's fine then

**Pull request closed without merging**